lib/mayaUsd/ufe/Global.cpp

```diff
@@ -178,8 +176,11 @@ MStatus initialize()
 #endif
 
 #ifdef UFE_V2_FEATURES_AVAILABLE
+    UsdUfe::Handlers          usdUfeHandlers;
     Ufe::RunTimeMgr::Handlers handlers;
-    handlers.hierarchyHandler = UsdHierarchyHandler::create();
+#ifdef UFE_V3_FEATURES_AVAILABLE
+    usdUfeHandlers.hierarchyHandler = MayaUsdHierarchyHandler::create();
+#endif
```

Comment:
Only need this one in Ufe v3 (or greater). Otherwise the UsdHierarchyHandler registered by UsdUfe is fine.